data/mods/gen1/moves.js

```diff
@@ -204,10 +201,7 @@ let BattleMovedex = {
 		self: {
 			volatileStatus: 'partialtrappinglock',
 		},
-		onBeforeMove(pokemon, target, move) {
-			// Removes must recharge volatile even if it misses
-			target.removeVolatile('mustrecharge');
-		},
+		// FIXME: onBeforeMove(pokemon, target) {target.removeVolatile('mustrecharge')}
```

Comment:
What's going on here?

Reply:
There is no ``singleEvent('BeforeMove', effect, effectData, move)``

Reply:
Oh, so this always did nothing? Huh.